src/main/java/com/visenze/datatracking/DataTracking.java

```diff
@@ -0,0 +1,164 @@
+package com.visenze.datatracking;
+
+
+
+import android.content.Context;
+import android.util.Log;
+
+import com.visenze.datatracking.data.DataTrackingResponse;
+import com.visenze.datatracking.data.DeviceData;
+import com.visenze.datatracking.data.EventData;
+import com.visenze.datatracking.data.EventsBody;
+import com.visenze.datatracking.data.ResultData;
+import com.visenze.datatracking.utils.Utils;
+
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+import java.util.UUID;
+
+
+import retrofit2.Call;
+import retrofit2.Callback;
+import retrofit2.Response;
+
+public class DataTracking {
+
+    public static final String TAG = "DataTracking";
+    public static final long SESSION_TIMEOUT = 900000; //15 min
```

Comment:
default session timeout is 30 minutes, not 15 mins

Reply:
changed